code/modules/mob/living/simple_animal/hostile/pirate.dm

```diff
@@ -50,6 +50,8 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	speed = 1
+	loot = list(/obj/effect/mob_spawn/human/corpse/pirate/spacepirate,
+			/obj/item/weapon/melee/transforming/energy/sword/pirate)

```

Comment:
Bit uhhh, OP

Reply:
The energy cutlass and laser gun were already standard loot drops on those mobs. I can remove them if needed.